phpcs.xml.dist

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ruleset name="CakePHP Core">
+ <rule ref="./vendor/cakephp/cakephp-codesniffer/CakePHP/ruleset.xml"/>
```

Comment:
I hate space indentation. It is soo wrong.
But in this case, would maybe 2 or 4 at least be less wrong than just 1? :)

Reply:
It's a copy-paste from the core repo.

Reply:
Part of the problem of space usage. I guess that should also be corrected to what is configured for xml files (the .dist just by addicent didnt get any support here when it was edited, thus it happened).

Reply:
Correct that if you please. 

My doctor told me today not to worry about irrelevant stuff. It's bad for your health :)

Reply:
haha, then you probably have a complete PR here that should be closed :) Not too valid an argument here IMO.

Reply:
Alt+Shift+F did some automagic here. Is that OK now?